docs/deploying/azure.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /docs/deploying/azure/index.html
+layout: docs
+---
+
+If you've been following along with [Getting Started](../README.md), it's time to deploy so you can use it beyond just your local machine.
+[Azure](http://http://azure.microsoft.com/) is a somewhat easy and probably unsupported way to deploy hubot.
+
+You will need to install the azure-cli via npm after you have follow the initial instructions for your hubot.
+
+    % npm install -g azure-cli
+
+Inside your new hubot directory, make sure you've created a git repository, and that your work is committed:
+
+    % git init
+    % git add .
+    % git commit -m "Initial commit"
+
+Then create a GitHub repository for your hubot. This is where zzure will pull your code from instead of needing to deploy directly from your dev machine to Azure.
+
+    % git remote add origin _your GitHub repo_
+	% git push -u origin master
+	
+Create a linked Azure website. In Azure, create a website and select integrated source control. When it asks "where is your source control" select GitHub and link this website to your git repo that you created in the previous step. If you have downloaded the Azure PowerShell modules, you can also do this via PowerShell.
+
+    % $creds = Get-Credential
+	% New-AzureWebsite mynewhubot -github -githubrepository yourgithubaccount/yourhubotreponame -githubcredentials $creds
+	
+Once you have done this, Azure will deploy your site any time you commit and push to GitHub. Your hubot won't run quite right yet, though. Next, you need to configure the deployment to tell Azure how to run hubot.
+	
+First, run the follow command to add `deploy.cmd` to your hubot directory. This is the file that Azure uses to know how to deploy your node application.
+
+    % azure site deploymentscript --node
+	
+Then, edit this file and look for the sections that give you steps 1, 2 and 3. You're going to add a 4th step:
+
+    :: 4. Create Hubot file with a coffee extension
+    copy /Y “%DEPLOYMENT_TARGET%\node_modules\hubot\bin\hubot” “%DEPLOYMENT_TARGET%\node_modules\hubot\bin\hubot.coffee”
+
+Now, create a new file in the base directory of hubot called `server.js` and put these two lines into it:
+
+    require('coffee-script/register');
+	module.exports = require('hubot/bin/hubot.coffee');
+	
+Save this file. Then open up `external-scripts.json` and remove the following two lines, as these two bits aren't compatible with Azure.
+
+    "hubot-heroku-keepalive",
+	"hubot-redit-brain",
+	
+Save this file then install coffee-script as a requirement to your local hubot. Azure requires this to run hubot properly.
```

Comment:
hubot already depends on coffee-script. Were you running into problems without this?